src/components/Tab/TabContent/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+
+export interface TabContentProps {
+  name: string;
+  active?: boolean;
+  icon?: React.ComponentType<any>;
+  children?: ReactNode;
+}
+
+export const TabContent: React.FC<TabContentProps> = ({
+  children,
+  active,
+}: TabContentProps) => {
+  return (
+    <div className={active ? 'block' : 'hidden'} id="link1">
```

Comment:
We have an id encoded here, the id must be unique on a page, maybe you can set this id using a prop

Maybe we can define a default id for the components?

Example:
- String assembly:
`<div id='ComponentName-${id}' ... />`

- Component usage:
`<TabContent id="link1" ... />`

- Rendered:
`<div id="TabContent-link1" ... />`